src/log/Logger.ts

```diff
@@ -126,11 +126,13 @@ class Logger {
                 const componentCount = config.getComponents().length;
                 const validComponentCount =
                     config.getComponents().filter(current => doesComponentExist(rootFolderPath, current.path)).length;
+                const zeplinComponentDescriptors = config.getAllZeplinComponentDescriptors();
                 logHeaders.push(
                     `Project count: ${validProjectCount}/${projectCount}`,
                     `Styleguide count: ${validStyleguideCount}/${styleguideCount}`,
                     `Component count: ${validComponentCount}/${componentCount}`,
-                    `Zeplin Component count: ${config.getAllZeplinComponentNames().length}`,
+                    `Zeplin Component Id count: ${zeplinComponentDescriptors.zeplinIds?.length ?? 0}`,
```

Comment:
```suggestion
                    `Zeplin Component ID count: ${zeplinComponentDescriptors.zeplinIds?.length ?? 0}`,
```

Reply:
it'd better be "id" regarding my other comments